packages/mongodb-chatbot-server/src/routes/responses/createResponse.ts

```diff
@@ -233,6 +232,13 @@ export function makeCreateResponseRoute({
         });
       }
 
+      dataStreamer.connect(res);
+
+      // TODO: stream a created message
+      dataStreamer.streamResponses({
```

Comment:
there'll be a bunch of additional data you pass here, right?

Reply:
yup! on it